doc/Client-Portal Protocol.md

```diff
@@ -90,7 +91,19 @@ This a format for client reconnects.
 
 **RequestName**: "relogin"<br>
 
-**RequestData**: The ReconnectionTicket object defined in 3.3.1 section.<br>
+**RequestData**: The ReconnectionTicket object defined in 3.3.1 section and PendingMessages:
+
+  object(ReloginResponse)::
+    {
+      ticket: string(Base64Encoded(object(ReconnectionTicket)))
+      messages: [
```

Comment:
Please describe `messages`, at least let developers who don't use our client SDKs know what kind of messages should be sent to server.

Reply:
Refer to the "NotificationName" and "NotificationData" in this document